src/video/canvas.cpp

```diff
@@ -264,17 +264,19 @@ Canvas::draw_gradient(const Color& top, const Color& bottom, int layer,
 
 void
 Canvas::draw_filled_rect(const Rectf& rect, const Color& color,
-                         int layer)
+                         int layer, const Blend& blend)
```

Comment:
This argument should be removed, since:

1. It's not passed, so it's unneeded.
2. Even if it was passed, it'd be better to use the other, more advanced `Canvas::draw_filled_rect()` overload for providing a blend.